classes/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Kindly make sure that the `Book` class has a method for adding rentals, so the `many-to-many` relationship between `Person` and `Book` is set up with the intermediate class `Rental`

```ruby
def add_rental(person, date)
...
...
```